src/content/en/2020/resource-hints.md

```diff
@@ -5,19 +5,367 @@ chapter_number: 21
+<link rel="preload" href="desktop.css" as="style" media=”only screen and (max-width: 768px)”>
+<link rel="preload" href="mobile.css" as="style" media=”~~~”>
+```
+
+The most popular media value used with `preload` is a combination of `only screen and (max-width: 768px)` and `screen and (min-width: 767px)`. Seeing over 2,100 different combinations of media queries in the 2020 dataset encourages us to consider how wide the variance is between concept and implementation of responsive design from site to site.
+
+### Best practices
+
+Using Resource Hints can be confusing at times, so let's go over some quick best practices to follow based on Lighthouse's automated audit.
+
+To safely implement `dns-prefetch` and `preconnect` make sure to have them in separate links tags.
+
+```html
+<link rel="preconnect" href="http://example.com">
+<link rel="dns-prefetch" href="http://example.com">
+```
+
+Implementing a `dns-prefetch` fallback in the same `<link>` tag causes a [bug](https://bugs.webkit.org/show_bug.cgi?id=197010) in Safari that cancels the `preconnect` request.
+
+
+{{ figure_markup(
+  caption="Resource hints on desktop that use either preconnect or dns-prefetch use both in the same hint.",
+  content="1.93%",
+  classes="big-number",
+  sheets_gid="281984550",
+  sql_file="preconnect_and_dnsprefetch_relative.sql"
+) }}
+
+Close to 2% of pages (~40k) reported the issue of both `preconnect` & `dns-prefetch` in a single resource.
+
+{{ figure_markup(
+  caption="Pages that pass the preconnect Lighthouse audit",
+  content="19.67%",
+  classes="big-number",
+  sheets_gid="152449420",
+  sql_file="lighthouse_preconnect.sql"
+) }}
+
+We saw only 19.67% of pages passing Lighthouse’s "[Preconnect to required origins](https://web.dev/uses-rel-preconnect/)" audit, creating a large opportunity for thousands of websites to start using `preconnect` or `dns-prefetch` to establish early connections to important third-party origins.
+
+{{ figure_markup(
+  caption="Pages that pass the preload Lighthouse audit",
+  content="84.6%",
+  classes="big-number",
+  sheets_gid="1047875076",
+  sql_file="lighthouse_preload.sql"
+) }}
+
+Running Lighthouse’s "[Preload key requests](https://web.dev/uses-rel-preload/)" audit resulted in 84.6% of pages passing the test, which is an astonishing result. If you are looking to use `preload` for the first time, remember, fonts and critical scripts are a good place to start.
+
+### Native Lazy Loading
+
+Now let’s celebrate the first year of the [Native Lazy Loading](https://addyosmani.com/blog/lazy-loading/) API, which at the time of publishing already has over [72%](https://caniuse.com/loading-lazy-attr) browser support. This new API can be used to defer the load of below-the-fold iframes and images on the page until the user scrolls near them. This can reduce data usage, memory usage, and helps speed up above-the-fold content. Opting-in to lazy load is as simple as adding `loading=lazy`  on `<iframe>` or `<img>` elements.
```

Comment:
Worth mentioning that Safari, as the last major browser to support this, has implemented this but not turned it on yet at which point support (for images at least) will jump into the high 90s?

Reply:
Do you have a link we could use to mention this information?

Reply:
Can't find an official link (the [webkit bug](https://bugs.webkit.org/show_bug.cgi?id=196698) is marked as resolved

Let's just use caniuse: https://caniuse.com/loading-lazy-attr

Reply:
Sounds good!